eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -165,7 +162,7 @@ case class LiftChannelExclusion(desc: ChannelDesc)
 // @formatter:on
 
 // @formatter:off
-case class SendChannelQuery(remoteNodeId: PublicKey, to: ActorRef, flags_opt: Option[QueryChannelRangeTlv])
+case class SendChannelQuery(chainHash: ByteVector32, remoteNodeId: PublicKey, to: ActorRef, flags_opt: Option[QueryChannelRangeTlv])
```

Comment:
I think it would be nice to move all these objects that are currently defined directly in the router namespace to the objects they really belong to

Reply:
And there are also some helper functions that are still in the `Router` object that can move to the handler file they're used in.

Reply:
Done with ee49982.